src/storage/src/sink/kafka.rs

```diff
@@ -1053,7 +1006,7 @@ where
                 key_desc,
                 value_desc,
                 matches!(envelope, Some(SinkEnvelope::Debezium)),
-                connection.consistency.is_some(),
+                true,
```

Comment:
Do we ever create these with `false`? oh maybe this is an example of what you were talking about when you said you didn't do all the cleanup here

Reply:
Right! Hopefully this one gets eliminated in https://github.com/MaterializeInc/materialize/pull/14678